### PR TITLE
Add test for render-blocking with encoding

### PR DESCRIPTION
**Pull request by noamr**

### CODE REVIEW

File: html/dom/render-blocking/element-render-blocking-encoding.tentative.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+<link rel="expect" href="#ß" blocking="render">
+<script>
+promise_test(async t => {
+  await new Promise(resolve => {
+    requestAnimationFrame(() => {
+      t.step(() => assert_true(!!document.getElementById("ß"), "should block until eszett element"));
+      t.step(() => assert_false(!!document.getElementById("last"), "should not block until last element"));
+      resolve();
+    });
+  });
```
Comment (domenic) [inline html/dom/render-blocking/element-render-blocking-encoding.tentative.html:17]:
```suggestion
async_test(t => {
  requestAnimationFrame(t.step_func_done(() => {
    assert_not_equals(document.getElementById("ß"), null, "should block until eszett element");
    assert_equals(document.getElementById("last"), null, "should not block until last element");
  }));
```

**Review by domenic — COMMENTED** [review]
I'm not sure this test would fail if you changed to `<meta charset="utf-8">`... what is it really testing?

**Comment by domenic** [comment]
I realized that encoding only impacts the query, not the fragment. So I think tests are probably not necessary. (In theory, you could test the `<base>` case where the `<base>` URL has a non-ASCII query... but I don't think we need that level of coverage here.)

**Comment by noamr** [comment]
This test is nonsensical and the feature might change anyway.

**Pull request closed without merging**